src/utils/getCategoriesOption.js

```diff
@@ -1,8 +1,9 @@
 const getCategoriesOptions = (categories) => {
-  if (categories) {
+  if (Array.isArray(categories)) {
     return categories.map((item) => (item === 'null' ? null : item))
-  } else {
-    return
+  } else if (typeof categories === 'string') {
+    return categories === 'null' ? [null] : [categories]
   }
```

Comment:
const getCategoriesOptions = (categories) => {
  if (Array.isArray(categories)) {
    return categories.map((item) => (item === 'null' ? null : item))
  } else if (typeof categories === 'string') {
    return categories === 'null' ? [null] : [categories]
  } else {
    return []
  }
} 
<img width="1367" alt="Screenshot 2024-10-24 at 19 22 51" src="https://github.com/user-attachments/assets/94934c49-b29b-49d1-a222-97aeefe8c206">.

Reply:
when I added return [], on initial loading the questions don't show at all, they start showing if we have already selected some categories
<img width="1372" alt="Screenshot 2024-10-24 at 19 27 22" src="https://github.com/user-attachments/assets/2b61e82c-9305-4691-86b5-79328711e133">

Reply:
if (typeof categories === 'string')
in this case categories cannot be null (because typeof null is 'object'
Also, if we have return in previous condition, the else word is redundant

Reply:
https://github.com/user-attachments/assets/ade88e48-0d6b-4e3e-aba4-fe54eaac4b0e